lib/recipe.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: false
 # rubocop:disable Layout/SpaceAroundOperators
 
-#  Copyright 2019 Chef Software, Inc.
+#  Copyright 2019-2019, Chef Software Inc.
```

Comment:
?

Reply:
that's my vim script to auto-update copyright days

Reply:
@lamont-granquist 2019 - 2019 is weird though, yeah?

Reply:
yeah, but i haven't dug up why its doing that

Reply:
haha

Reply:
yeah i thought it used to handle single years but it very clearly doesn't now...